src/Passes/examples/ClassicalIrCommandline/README.md

```diff
@@ -19,18 +19,12 @@ whereas the latter is generated by executing:
 
 This generates a nice and short IR which makes not too overwhelming to understand what is going on.
 
-## Legacy passes
```

Comment:
Is legacy passes here referring to the pass manager? If so, are all passes now currently written only for the new pass manager? I saw an include for the legacy pass manager further above I believe, so the answer I assume is no.

Reply:
This section can go. We only use the new pass manager.

Reply:
Are the passes themselves at this time only written for the new pass manager, or is it just the tool integration that does not enable using the legacy manager? I saw commented out code related to the legacy pass manager in the tool and I saw a (not commented out) include for the legacy pass manager - is that still needed?

Reply:
We only use the new pass manager at this point in time.